packages/application/src/MetadataService.ts

```diff
@@ -14,35 +14,16 @@
  * limitations under the License.
  */
 
-import { Dialog, showDialog } from '@jupyterlab/apputils';
-import * as React from 'react';
-
 import { IDictionary } from './parsing';
-import { RequestHandler } from './requests';
+import { RequestHandler } from './RequestHandler';
 
 const ELYRA_SCHEMA_API_ENDPOINT = 'elyra/schema/';
 const ELYRA_METADATA_API_ENDPOINT = 'elyra/metadata/';
 
 /**
  * A service class for accessing the elyra api.
  */
-export class FrontendServices {
-  /**
-   * Displays a dialog for error cases during metadata calls.
-   *
-   * @param namespace - the metadata namespace that was being accessed when
-   * the error occurred
-   *
-   * @returns A promise that resolves with whether the dialog was accepted.
-   */
-  static noMetadataError(namespace: string): Promise<Dialog.IResult<any>> {
-    return showDialog({
-      title: 'Error retrieving metadata',
-      body: <p>No {namespace} metadata has been configured.</p>,
-      buttons: [Dialog.okButton()]
-    });
-  }
-
+export class MetadataService {
```

Comment:
this file should be `metadata.ts`

Reply:
I though we had decided to use the standard react naming convention of naming a file after the class it contains, even for utils. (We discussed in one of Nick's PRs last month I believe and decided to update them as we work on them rather than doing a refactor) I took the chance to update the naming convention for these since I was already changing the signature anyway. I'm ok changing them back to `requests` and `metadata` if you still think we should, I don't really have strong opinions on this.

Reply:
Where is that documented? I still see this https://github.com/elyra-ai/elyra/issues/917 open. I am really looking for consistency, either we document and make sure we are using what is documented or not, but I really don't want to change one by one and have a mess.

Reply:
understandable, and it's possible I'm remembering a verbal conversation at a scrum rather than one on a PR or Issue. At least for now I'll update this to leave the naming as is and we can revisit the naming issue after we return from the holidays